<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Experiment.qrc">
    <normaloff>:/latex/icons/FuncIcon2.png</normaloff>:/latex/icons/FuncIcon2.png</iconset>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitterHorizontal">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="filenameComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="maxVisibleItems">
             <number>100</number>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::InsertAtTop</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="removeCurrent">
            <property name="toolTip">
             <string>remove Current entry from Combo Box</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="Experiment.qrc">
              <normaloff>:/openicon/icons/edit-clear-locationbar-rtl.png</normaloff>:/openicon/icons/edit-clear-locationbar-rtl.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="statusLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitterVertical">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionNew"/>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionNew"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Files</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeWidget" name="fileTreeWidget">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Documentation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTreeWidget" name="docTreeWidget">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideRight</enum>
       </property>
       <property name="indentation">
        <number>10</number>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="windowTitle">
    <string>Evaluation Tests</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="addEvalRow">
         <property name="toolTip">
          <string>Add Row</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="Experiment.qrc">
           <normaloff>:/openicon/icons/edit-add.png</normaloff>
           <normalon>:/openicon/icons/edit-add.png</normalon>:/openicon/icons/edit-add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="removeEvalRow">
         <property name="toolTip">
          <string>Remove Row(s)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="Experiment.qrc">
           <normaloff>:/openicon/icons/edit_remove.png</normaloff>
           <normalon>:/openicon/icons/edit_remove.png</normalon>:/openicon/icons/edit_remove.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="lineWidth">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionStartScript">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/kde/icons/media-playback-start-7.png</normaloff>:/kde/icons/media-playback-start-7.png</iconset>
   </property>
   <property name="text">
    <string>Start Script</string>
   </property>
   <property name="toolTip">
    <string>Start the script</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionStopScript">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/kde/icons/media-playback-stop-7.png</normaloff>:/kde/icons/media-playback-stop-7.png</iconset>
   </property>
   <property name="text">
    <string>Stop Script</string>
   </property>
   <property name="toolTip">
    <string>Stop the script</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionStopScriptAndScan">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/openicon/icons/road-sign-us-stop.png</normaloff>:/openicon/icons/road-sign-us-stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop Script and Scan</string>
   </property>
   <property name="toolTip">
    <string>Stop the script and scan</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+X</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/openicon/icons/document-open-5.png</normaloff>:/openicon/icons/document-open-5.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/kde/icons/filesave.png</normaloff>:/kde/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/openicon/icons/document-revert-5.png</normaloff>:/openicon/icons/document-revert-5.png</iconset>
   </property>
   <property name="text">
    <string>Reset Editor to File</string>
   </property>
   <property name="toolTip">
    <string>Reset editor to file</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/other/icons/document-new-icon.png</normaloff>:/other/icons/document-new-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPauseScript">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/openicon/icons/media-playback-pause-7.png</normaloff>:/openicon/icons/media-playback-pause-7.png</iconset>
   </property>
   <property name="text">
    <string>Pause Script</string>
   </property>
   <property name="toolTip">
    <string>Pause the script</string>
   </property>
  </action>
  <action name="actionPauseScriptAndScan">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/openicon/icons/media-playback-pause-3.png</normaloff>:/openicon/icons/media-playback-pause-3.png</iconset>
   </property>
   <property name="text">
    <string>Pause Script and Scan</string>
   </property>
   <property name="toolTip">
    <string>Pause the script and scan</string>
   </property>
  </action>
  <action name="actionShowConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Console</string>
   </property>
  </action>
  <action name="actionShowDocs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Documentation</string>
   </property>
   <property name="toolTip">
    <string>Show Docs</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/kde/icons/gear.png</normaloff>:/kde/icons/gear.png</iconset>
   </property>
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Experiment.qrc"/>
 </resources>
 <connections/>
</ui>
